src/gym_electric_motor/physical_systems/electric_motors/dc_series_motor.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+
+from .dc_motor import DcMotor
+
+
+class DcSeriesMotor(DcMotor):
+    """The DcSeriesMotor is a DcMotor with an armature and exciting circuit connected in series to one input voltage.
+
+    =====================  ==========  ============= ===========================================
+    Motor Parameter        Unit        Default Value Description
+    =====================  ==========  ============= ===========================================
+    r_a                    Ohm         16e-3         Armature circuit resistance
+    r_e                    Ohm         48e-3         Exciting circuit resistance
+    l_a                    H           19e-6         Armature circuit inductance
+    l_e                    H           5.4e-3        Exciting circuit inductance
+    l_e_prime              H           1.7e-3        Effective excitation inductance
+    j_rotor                kg/m^2      0.025         Moment of inertia of the rotor
+    =====================  ==========  ============= ===========================================
+
+    =============== ====== =============================================
+    Motor Currents  Unit   Description
+    =============== ====== =============================================
+    i               A      Circuit current
+    =============== ====== =============================================
+    =============== ====== =============================================
+    Motor Voltages  Unit   Description
+    =============== ====== =============================================
+    u               V      Circuit voltage
+    =============== ====== =============================================
+
+    ======== ===========================================================
+    Limits / Nominal Value Dictionary Entries:
+    -------- -----------------------------------------------------------
+    Entry    Description
+    ======== ===========================================================
+    i        Circuit Current
+    omega    Angular Velocity
+    torque   Motor generated torque
+    u        Circuit Voltage
+    ======== ===========================================================
+    """
+
+    HAS_JACOBIAN = True
+    I_IDX = 0
+    CURRENTS_IDX = [0]
+    CURRENTS = ["i"]
+    VOLTAGES = ["u"]
+
+    # Motor parameter, nominal values and limits are based on the following DC Motor:
+    # https://www.heinzmann-electric-motors.com/en/products/dc-motors/pmg-132-dc-motor
+    _default_motor_parameter = {
+        "r_a": 16e-3,
+        "r_e": 48e-3,
+        "l_a": 19e-6,
+        "l_e_prime": 1.7e-3,
+        "l_e": 5.4e-3,
+        "j_rotor": 0.0025,
+    }
+    _default_nominal_values = dict(omega=300, torque=16.0, i=97, i_a=97, i_e=97, u=60, u_a=60, u_e=60)
+    _default_limits = dict(omega=400, torque=38.0, i=210, i_a=210, i_e=210, u=60, u_a=60, u_e=60)
```

Comment:
Should one specify all these values as floats by putting ".0" behind them?

(Applies to several files)